src/main/java/Reporter.java

```diff
@@ -11,7 +11,39 @@
  */
 public class Reporter {
     public static final String REPORT_SUFFIX = "_Report.txt";
+    public static final int REPORT_ALL          = 0xFFFFFFFF;
+    public static final int NUM_PROJECTS        = 0b1;
+    public static final int PROJECT_HEADERS     = 0b10;
+    public static final int SPRITES_PER_PROJECT = 0b100;
+    public static final int SPRITE_HEADERS      = 0b1000;
+    public static final int SCRIPTS_PER_SPRITE  = 0b1_0000;
+    public static final int SCRIPT_HEADERS      = 0b10_0000;
+    public static final int SCRIPT_LENGTHS      = 0b100_0000;
     private static final String TAB = "    ";
+    private int whatToReport;
```

Comment:
So, this implementation essentially has 32 "feature flags" that can turn on and off different report components? I suppose that will likely be enough. Or are `int`s 64 bits in java now?